File: CPP/Car.cpp
#pragma once

#include <string>
#include "Vehicle.cpp"

using namespace std;

class Car : public Vehicle {
    private:
        int car_doors;
        int seat;

    public:
        Car() {
            this->car_doors = -1;
            this->seat = -1;
        }
        Car(int car_doors, int seat, string license_plate, string brand, int prod_year, string color) : Vehicle(license_plate, brand, prod_year, color){
            this->car_doors = car_doors;
            this->seat = seat;
        }

        int getCarDoors() { return this->car_doors; }
        int getCarSeats() { return this->seat; }

        void setCarDoors(int car_doors) { this->car_doors = car_doors; }
        void setCarSeats(int seat) { this->seat = seat; }
};

File: Python/Car.py
from Vehicle import Vehicle # Imports Vehicle Class

class Car(Vehicle):
    # Attribute Declaration
    __car_doors = -1;
    __car_seat = -1;


    # Constructor
    def __init__(self, car_doors=-1, car_seat=-1,  license_plate="n/a", brand="n/a", prod_year=-1, color="n/a"):
        super().__init__(license_plate, brand, prod_year, color)
        self.__car_doors = car_doors
        self.__car_seat = car_seat = car_seat

    # Getter-Setter
    # Get Methods
    def getCarDoors(self):
        return self.__car_doors
    def getCarSeats(self):
        return self.__car_seat

    # Set Methods
    def setCarDoors(self, car_doors):
        self.__car_doors = car_doors
    def setCarSeats(self, car_seat):
        self.__car_seat = car_seat